README.md

```diff
@@ -67,6 +67,9 @@ config :libcluster,
         # The default service for children endpoints specifications.
         service: [name: "service_name"],
 
+        # Apply filtering
+        filter: ~s("canary" in Service.Tags),
```

Comment:
Just curious if Consul accepts multiple filters or just a single one (and how that would like).

Reply:
It does, you can combine filters by using logical operators in the filter parameter so it would be something like this:
```
filter: ~s("canary" in Service.Tags and not "staging" in Service.Tags)
```
https://developer.hashicorp.com/consul/api-docs/features/filtering#connecting-expressions